rest-api-spec/src/main/resources/rest-api-spec/api/delete.json

```diff
@@ -23,10 +23,9 @@
         }
       },
       "params": {
-        "consistency": {
-          "type" : "enum",
-          "options" : ["one", "quorum", "all"],
-          "description" : "Specific write consistency setting for the operation"
+        "wait_for_active_shards": {
+          "type" : "string",
+          "description" : "Explicit active shard count required for the operation"
```

Comment:
can we copy over the descriptions from the java docs? we should make it clear this is required for the operation to start.